### ISSUE TITLE
x/website: A11y, links need discernible text 

### ISSUE THREAD

**Issue by A11Ybot-OP** [golang/go#53565]
website https://go.dev/
Title: WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (a[href$="google/"])
Tags: Accessibility, WCAG 2.4.4, WCAG 4.1.2, link-name

Issue: Ensures links have discernible text (link-name - https://accessibilityinsights.io/info-examples/web/link-name)

Target application: The Go Programming Language - https://go.dev/

Element path: a[href$="google/"]

Snippet: `<a href="/solutions/google/" class="WhoUsesCaseStudyList-caseStudyLink"> <img loading="lazy" height="48" width="30%" src="/images/logos/google.svg" class="WhoUsesCaseStudyList-logo" alt=""> <p>View case study</p> </a>`

How to fix: 
Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Chrome version 102.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
